client/marketing/components/knowledge-base/placeholder.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const KnowledgebaseCardPostPlaceholder = ( index ) => {
+	const classNameBase = 'woocommerce-marketing-knowledgebase-card__post';
+
+	return (
+		<div
+			className={ 'is-loading ' + classNameBase }
+			key={ index }
+			aria-hidden="true"
+		>
+			<div className={ classNameBase + '-img is-placeholder' }></div>
+			<div className={ classNameBase + '-text' }>
+				<h3 className="is-placeholder"></h3>
```

Comment:
JS tests are flagging a couple of errors for the headings `17:5  error  Headings must have content and the content must be accessible by a screen reader  jsx-a11y/heading-has-content`

Wonder if we can ignore or use another element like a div here? I think the inbox panel might use a div instead of a heading for this

Reply:
Sorry, I forgot to push recent changes 😶 

I was thinking of switching to `div`, but that would require changing CSS, adding more classes, and making it less semantic. AFAIK, for a11y perspective setting `aria-hidden="true"` on any ancestor should do the job - make it hidden for a11y tools. However, for this check, it seems to be not enough. I've added `aria-hidden="true"` explicitly on the `h*` elements to make it happy.

Reply:
Nice, TIL